mreg/types.py

```diff
@@ -0,0 +1,5 @@
+from enum import Enum
+
+class IPAllocationMethod(Enum):
```

Comment:
There is a case to be made for using `StrEnum` here, since we already build our container image for Python 3.11. `StrEnum` helps avoid bugs/visual noise from forgetting to use `.value` when displaying and comparing such Enums.

However, we still officially support 3.10, so yeah... This is fine for now.

Reply:
Yeah, I poked at StrEnum, but 3.10. :(